staging/src/k8s.io/apiserver/pkg/apis/audit/v1/types.go

```diff
@@ -72,17 +72,27 @@ const (
 type Event struct {
 	metav1.TypeMeta `json:",inline"`
 
-	// AuditLevel at which event was generated
+	// Audit level at which event was generated. Valid values include: 'None',
+	// 'Metadata', 'Request' and 'RequestResponse'.
 	Level Level `json:"level" protobuf:"bytes,1,opt,name=level,casttype=Level"`
 
 	// Unique audit ID, generated for each request.
 	AuditID types.UID `json:"auditID" protobuf:"bytes,2,opt,name=auditID,casttype=k8s.io/apimachinery/pkg/types.UID"`
 	// Stage of the request handling when this event instance was generated.
+	// The value can be one of:
+	// - 'RequestReceived': The stage for events generated as soon as the audit handler receives
+	//   the request, and before it is delegated down the handler chain.
+	// - 'ResponseStarted': The stage for events generated once the response headers are sent,
+	//   but before the response body is sent. This stage is only generated for long-running requests
+	//   (e.g. watch).
+	// - 'ResponseComplete': The stage for events generated once the response body has been completed,
+	//   and no more bytes will be sent.
+	// - 'Panic': The stage for events generated when a panic occurred.
```

Comment:
there was a concern about using markdown in API doc: https://github.com/kubernetes/kubernetes/pull/109259#issuecomment-1117099014. I want to make sure it's on your radar

Reply:
I'm aware of that comment, but I don't agree with it. For OpenAPI 2.0, it is said that "for description, ([GFM syntax](https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown) can be used for rich text representation)". For OpenAPI 3.0, it is said that "Throughout the specification description fields are noted as supporting CommonMark markdown formatting.".

Reply:
I agree with **OpenAPI** using Markdown. I'm not convinced that as a project we have committed to treating _code comments_ as Markdown.

**This looks OK to me**: the single quotes are appropriate here for a sort of halfway house between a commitment to Markdown and not.

I would like to see some specially marked doc comments treated (and linted) as Markdown; maybe one day I'd have the cycles for a KEP, and the project could have the cycles to adopt it. Plenty of of work there, too.